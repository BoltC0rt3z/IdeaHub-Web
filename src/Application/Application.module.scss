.content {
  margin-top: 80px;
}

.header {
  position: fixed;
  width: 100%;
  z-index: 1;
}
