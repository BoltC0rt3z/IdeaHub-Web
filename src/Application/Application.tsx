import React from "react";
import { BrowserRouter as Router, Switch, Route } from "react-router-dom";
import { Row, Col } from "antd";

import { SideMenu } from "../SideMenu";
import { PageHeader } from "../Components";

import styles from "./Application.module.scss";

const Application: React.FC = () => {
  return (
    <Router>
      <Row type="flex" className={styles.application}>
        <Col span={4}>
          <SideMenu />
        </Col>
        <Col span={20}>
          <PageHeader />
          <Switch>
            <Route exact path="/ideahub/" />
          </Switch>
        </Col>
      </Row>
    </Router>
  );
};

export default Application;
