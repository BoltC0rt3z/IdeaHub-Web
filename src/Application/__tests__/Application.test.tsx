import React from "react";
import { mount, ReactWrapper } from "enzyme";

import Application from "../Application";

describe("Tests for application component", () => {
  const wrapper: ReactWrapper = mount(<Application />);

  it("Should render application component without crushing", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
