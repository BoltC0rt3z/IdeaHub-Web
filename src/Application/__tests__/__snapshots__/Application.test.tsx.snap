// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for application component Should render application component without crushing 1`] = `ReactWrapper {}`;
